compiler/rustc_privacy/src/lib.rs

```diff
@@ -112,13 +113,26 @@ where
                 ty.visit_with(self)
             }
             ty::PredicateKind::RegionOutlives(..) => ControlFlow::CONTINUE,
-            ty::PredicateKind::ConstEvaluatable(..)
+            ty::PredicateKind::ConstEvaluatable(_, substs)
```

Comment:
this doesn't quite seem right here
```suggestion
ty::PredicateKind::ConstEvaluatable(def, substs)
    if self.def_id_visitor.tcx().features().const_evaluatable_checked => {
    if let Ok(Some(ct)) = AbstractConst::new(tcx, def, substs) {
        const_evaluatable::walk_abstract_const(tcx, ct, |node| match node {
            ACNode::Leaf(leaf) => {
                let leaf = leaf.subst(tcx, ct.substs);
                self.visit_const(leaf)
            }
            ACNode::Binop(..)
            | ACNode::UnaryOp(..)
            | ACNode::FunctionCall(_, _) => ControlFlow::CONTINUE,
        })?;
    }
}

```